typescript/web/src/hooks/use-user.tsx

```diff
@@ -78,6 +78,7 @@ const useWorkspaceProvider = () => {
     () => workspaces?.find(({ slug }) => slug === workspaceSlug),
     [workspaceSlug, workspaces]
   );
+  // FIXME: That 'if' is causing problems on workspace creation and deletion
```

Comment:
Thanks for the info @Jordanlelay 👍 Let's fix it before merging this PR and remove the comment once done 🚀

Reply:
After re-reading the PR, the comment does not adds any information. Indeed it's this line which throws the error since it's where the message is coming from. But it does not adds any information about why there is no workspace with the newly created slug.

Can you please remove the comment?